config/base.yml

```diff
@@ -111,6 +111,16 @@ before_script:
   # The following line is used for testing the cms module itself
   - if [[ $BEHAT_TEST && $cms_installed == 0 ]]; then (vendor/bin/serve --bootstrap-file tests/behat/serve-bootstrap.php &> artifacts/serve.log &); fi
 
+  # codecov
+  # https://about.codecov.io/blog/introducing-codecovs-new-uploader/
+  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then curl https://keybase.io/codecovsecurity/pgp_keys.asc | gpg --import; fi
+  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then curl -Os https://uploader.codecov.io/latest/codecov-linux; fi
+  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then curl -Os https://uploader.codecov.io/latest/codecov-linux.SHA256SUM; fi
+  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then curl -Os https://uploader.codecov.io/latest/codecov-linux.SHA256SUM.sig; fi
+  - if [[ $PHPUNIT_COVERAGE_TEST ]]; then gpg --verify codecov-linux.SHA256SUM.sig codecov-linux.SHA256SUM; fi
```

Comment:
`gpg` and `shasum` aren't mentioned as included on xenial, I guess we'll find out when travis runs.
https://docs.travis-ci.com/user/reference/xenial/